redlite/metric/_basic.py

```diff
@@ -2,9 +2,9 @@
 from .util import normalize_string
 
 
-class PrefixMetric(NamedMetric):
+class MatchMetric(NamedMetric):
```

Comment:
Subclassing does not work nicely with autodoc, sadly. Constructor docs get lost.

Maybe the best would be to not use subclassing at all, and instead do composition?

In any case, please make sure that API docs are generated properly.